examples/dns/record.py

```diff
@@ -0,0 +1,26 @@
+from cloudflare import Cloudflare
+
+
+# define your variable here
+
+cloudflare_token = "your-cloudflare-token"
+cloudflare_zone_id = "zone"
+alias_name = "www.mydns.com"
+cannonical_name = "cname.example.com"
```

Comment:
let's move these inline to the kwargs as using variables doesn't give us anything extra here.

Reply:
we can also use a similar pattern to https://github.com/cloudflare/cloudflare-python/blob/d04dfee45618904afb9569b0202758b8d6b63801/examples/workers/ai/demo.py#L6-L8 for the `zone_id` values.